src/grt/src/GlobalRouter.cpp

```diff
@@ -1218,7 +1217,8 @@ void GlobalRouter::computeUserLayerAdjustments(int max_routing_layer)
   const std::vector<int>& ver_capacities = grid_->getVerticalEdgesCapacities();
 
   for (int layer = 1; layer <= max_routing_layer; layer++) {
-    float adjustment = adjustments_[layer];
+    float adjustment
```

Comment:
Same here. Even if you only use it once, I think it's good to separate it for readability.